tools/ci/provision.sh

```diff
@@ -38,11 +38,13 @@ sudo apt-get --assume-yes install libc6:i386 libc6-dev:i386
 # libstdc++-10-dev was added for the sake of clang. see: https://stackoverflow.com/q/26333823/10278
 sudo apt-get --assume-yes install \
   build-essential \
+  clang \
   clang-format-19 \
   cmake \
   curl \
   graphviz \
   g++-11 \
+  gcovr \
```

Comment:
I intend to leave `gcovr` here in the `apt-get install` list.

As an aside (or "note to self"), I was also able to install (and get a specific version) by doing:
```
pip install git+https://github.com/gcovr/gcovr.git@87771a89336904e6c4a670b4096f8b8a814b8c22
```